cmd/skywire-peering-daemon/commands/root.go

```diff
@@ -0,0 +1,57 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	spd "github.com/SkycoinProject/skywire-peering-daemon/pkg/daemon"
+)
+
+var cfg *spd.Config
+
+var rootCmd = &cobra.Command{
+	Use:   "skywire-peering-daemon",
+	Short: "A skywire-peering-skywire-peering-daemon",
```

Comment:
Redundant description.

Reply:
Very true. Refactored it to a long description.